pathfinder/tests/fixtures/network_service.py

```diff
@@ -68,12 +76,100 @@ def populate_token_networks(
+def populate_token_networks_simple(
+    token_networks: List[TokenNetwork],
+    private_keys: List[str],
+    addresses: List[Address],
+    web3: Web3
+):
+    """ Initializes all token networks with the same default network, consisting of the channels
+    described in channel_descriptions.
+    The tuples in channel_descriptions define the following:
+    (
+        p1_index,
+        p1_deposit,
+        p1_transferred_amount,
+        p1_fee,
+        p2_index,
+        p2_deposit,
+        p2_transferred_amount,
+        p2_fee
+    )
+    Topology:
+          /-------------\
+    0 -- 1 -- 2 -- 3 -- 4    5 -- 6
+     \-------/
+    """
+    channel_descriptions = [
+        (0, 100,  20, 0.0010, 1,  50,  10, 0.0015),  # capacities  90 --  60
+        (1,  40,  10, 0.0008, 2, 130, 100, 0.0012),  # capacities 130 --  40
+        (2,  90,  10, 0.0007, 3,   0,   0, 0.0010),  # capacities  80 --  10
+        (3,  50,  20, 0.0011, 4,  50,  20, 0.0011),  # capacities  50 --  50
+        (0,  40,  40, 0.0015, 2,  80,   0, 0.0025),  # capacities   0 -- 120
+        (1,  30,  10, 0.0100, 4,  40,  15, 0.0018),  # capacities  35 --  35
+        (5, 500, 900, 0.0030, 6, 750, 950, 0.0040),  # capacities 550 -- 700
     ]
 
+    for channel_id, (
+        p1_index,
+        p1_deposit,
+        p1_transferred_amount,
+        p1_fee,
+        p2_index,
+        p2_deposit,
+        p2_transferred_amount,
+        p2_fee
+    ) in enumerate(channel_descriptions):
+        for token_network in token_networks:
+            token_network.handle_channel_opened_event(
+                channel_id,
+                addresses[p1_index],
+                addresses[p2_index]
+            )
+
+            token_network.handle_channel_new_deposit_event(
+                channel_id,
+                addresses[p1_index],
+                p1_deposit
+            )
+            token_network.handle_channel_new_deposit_event(
+                channel_id,
+                addresses[p2_index],
+                p2_deposit
+            )
+
+            p1_balance_proof = BalanceProof(
+                0,
+                p1_transferred_amount,
+                EMPTY_MERKLE_ROOT,  # TODO: include some pending locks here
+                channel_id,
+                token_network.address,
+                web3.net.version,
+                b'',
+                private_key=private_keys[p1_index]
+            )
+            p2_balance_proof = BalanceProof(
+                0,
+                p2_transferred_amount,
+                EMPTY_MERKLE_ROOT,  # TODO: include some pending locks here
+                channel_id,
+                token_network.address,
+                web3.net.version,
+                b'',
+                private_key=private_keys[p2_index]
+            )
+            token_network.update_balance(p1_balance_proof, [])
+            token_network.update_balance(p2_balance_proof, [])
+
+            p1_fee_msg = str(p1_fee).encode()
```

Comment:
A helper function for creating a fee message and signature pair would be nice for readability.

Reply:
True. Will add.